### ISSUE TITLE
cannot read some data

### ISSUE THREAD

**Issue by insaneazka** [NaturalIntelligence/fast-xml-parser#341]
* [ ] Are you running the latest version?
* [ ] Have you included sample input, output, error, and expected output? 
* [ ] Have you checked if you are using correct configuration?
* [ ] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)?

### Description
 
I tried and something was not detected by your library

I have circled undetected data ...

are these bugs? or can't read the data?
### Code
```js
var fetch =  require('node-fetch')
var xml = require('fast-xml-parser')
async function test() {
    var url = "https://data.bmkg.go.id/DataMKG/MEWS/DigitalForecast/DigitalForecast-Bali.xml"
    var response = await fetch(url).then(response => response.text())
    var document = await xml.parse(response);
    
    console.log(JSON.stringify(document, null, 2))
}
```

### Output

{
  "data": {
    "forecast": {
      "issue": {
        "timestamp": 20210507024021,
        "year": 2021,
        "month": 5,
        "day": 7,
        "hour": 2,
        "minute": 40,
        "second": 21
      },
      "area": [
        {
          "name": [
            "Amlapura",
            "Kab. Karangasem"
          ],
          "parameter": [
           This is very long, I'll just copy it here
}
### expected data

{
    "data": {
        "forecast": {
            "issue": {
                "timestamp": 20210507024021,
                "year": 2021,
                    "month": 5,
                    "day": 7,
                    "hour": 2,
                    "minute": 40,
                    "second": 21
            },
            "area": {
                "id": 501162,
                "latitude": -8.45,
                "description": "blabla", 
                "data": [
                    {
                        "name": [
                            "Amlapura",
                            "Kab. Karangasem"
                        ],
                        "parameter": [
                            "This is very long, I'll just copy it here"
                        ]
                    }
                ]
            }
        }
    }
}

**Would you like to work on this issue?**

- [X
![Screenshot (67)](https://user-images.githubusercontent.com/82513502/117442666-463ca480-af61-11eb-93b7-eb669be0926e.png)
] Yes
- [ ] No

[Bookmark](https://github.com/NaturalIntelligence/fast-xml-parser/stargazers) this repository for further updates.

**Comment by amitguptagwl** [comment]
You're using parser with default options. You should set necessary flags to read attributes. Please check README.md for more detail.
